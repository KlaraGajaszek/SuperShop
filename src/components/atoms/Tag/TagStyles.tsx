import styled from 'styled-components';

export const TagStyles = styled.div`
  margin: 30px 20px 0px 0;
  border: none;
  outline: none;
  cursor: pointer;
  background-color: #ffff;
  border-radius: 15px;
  padding: 5px 30px;
  background-color: rgba(144, 202, 249, 0.5);
  box-shadow: 0 8px 16px -8px rgba(0.8, 0.8, 0.8, 0.4);
`;
