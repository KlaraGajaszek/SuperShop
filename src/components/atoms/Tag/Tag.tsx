import React, { FC } from 'react';

import { TagStyles } from './TagStyles';

type TagProps = {
  title: string;
};

const Tag: FC<TagProps> = ({ title }) => {
  return <TagStyles data-testid="tag">{title}</TagStyles>;
};

export default Tag;
