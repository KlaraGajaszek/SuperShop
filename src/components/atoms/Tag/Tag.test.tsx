import React from 'react';
import renderer from 'react-test-renderer';
import Tag from './Tag';

describe('Tag', () => {
  it('renders correctly', () => {
    const tag = renderer.create(<Tag title="title" />).toJSON();
    expect(tag).toMatchSnapshot();
  });
});

describe('Tag', () => {
  it('renders correctly next function', () => {
    const tag = renderer.create(<Tag title="title" />).toJSON();
    expect(tag).toMatchInlineSnapshot();
  });
});
