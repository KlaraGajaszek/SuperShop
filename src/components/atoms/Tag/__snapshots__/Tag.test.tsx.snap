// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tag renders correctly 1`] = `
<div
  className="sc-bdnylx byWsuf"
>
  title
</div>
`;
